google/data_source_google_netblock_ip_ranges.go

```diff
@@ -0,0 +1,128 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+func dataSourceGoogleNetblockIpRanges() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleNetblockIpRangesRead,
+
+		Schema: map[string]*schema.Schema{
+			"cidr_blocks": {
+				Type:     schema.TypeList,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Computed: true,
+			},
+			"cidr_blocks_ip4": {
```

Comment:
For the sake of consistency with our other resources like cluster and route, let's call this field: `cidr_blocks_ipv4`. Same for the ipv6 block below.

Reply:
done.